xkit.js

```diff
@@ -128,889 +167,828 @@ XKit = {
 			if (!xkit_main.errors && xkit_main.script) {
 				XKit.console.add("Trying to run xkit_main.");
 				try {
-					/* jshint evil: true */
 					eval(xkit_main.script + "\n//# sourceURL=xkit/xkit_main.js");
+					XKit.frame_mode = true;
```

Comment:
It didn't change? 
![image](https://cloud.githubusercontent.com/assets/233815/21674077/c8ded27e-d2f1-11e6-9b8a-ee46e36eb92b.png)